crates/evm/evm/src/inspectors/stack.rs

```diff
@@ -433,7 +433,8 @@ impl InspectorStack {
         if let Some(config) = mode.into_config() {
             *self.tracer.get_or_insert_with(Default::default).config_mut() = config;
         } else {
-            self.tracer = None;
+            // initiate a dummy tracer that does nothing
+            self.tracer = Some(TracingInspector::new(TracingInspectorConfig::none()));
```

Comment:
yeah, I'd like to not have that too

do you have ideas how we could solve this differently? the issue is that if tracing is disabled at the moment of calling the cheatcode, then we can't just initialize it with new instance because it will panic on `call_end` for the cheatcode call.

we could consider adding helpers in `revm-inspectors`, and for now only support those cheats with tracing enabled. basically, tests would only pass with `-vvv`